doc/cephadm/install.rst

```diff
@@ -463,3 +465,27 @@ have access to all hosts that you plan to add to the cluster.
       cephadm --image *<hostname>*:5000/ceph/ceph bootstrap --mon-ip *<mon-ip>*
 
 .. _cluster network: ../rados/configuration/network-config-ref#cluster-network
+
+.. _cephadm-bootstrap-custom-ssh-keys:
+
+Deployment with custom SSH keys
+-------------------------------
+
+Bootstrap allows users to create their own private/public SSHkey pair
+rather than having cephadm generate them itself
```

Comment:
```suggestion
rather than having cephadm generate them itself.
```

Maybe even
```suggestion
rather than having cephadm generate them automatically.
```